dotnet-interactive/HttpRouting/LspRouter.cs

```diff
@@ -44,16 +44,31 @@ public async Task RouteAsync(RouteContext context)
                     // TODO: figure out how to switch kernels based on #!fsharp, etc.
                     if (targetKernel is CompositeKernel composite)
                     {
+                        await composite.RunDeferredCommandsAsync();
                         targetKernel = composite.ChildKernels.FirstOrDefault(k => k.Name == composite.DefaultKernelName);
                     }
 
                     if (targetKernel is KernelBase kernelBase)
                     {
+                        await kernelBase.RunDeferredCommandsAsync();
                         var methodName = segments[1];
                         var lspBodyReader = new StreamReader(context.HttpContext.Request.Body);
                         var stringBody = await lspBodyReader.ReadToEndAsync();
                         var request = JObject.Parse(stringBody);
-                        var response = await kernelBase.LspMethod(methodName, request);
+
```

Comment:
Same concern here: We should use a method that encapsulates which configuration will be used, rather than going directly to `Newtonsoft.Json`.